src/ProjectModule/Components/ProjectForm/ProjectForm.tsx

```diff
@@ -0,0 +1,138 @@
+import React, { useEffect, useState } from "react";
+import styles from "./ProjectForm.module.css";
+import { useNavigate, useParams } from "react-router-dom";
+import { useForm } from "react-hook-form";
+import axios from "axios";
+import { baseUrl } from "../../../Constants/Components/Urls";
+import { toast } from "react-toastify";
+import Loader from "../../../SharedMoudule/Components/Loading/Loading";
+
+export default function ProjectForm() {
+  const navigate = useNavigate();
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+    setValue,
+  } = useForm();
+  const token = localStorage.getItem("userToken");
+  const { action } = useParams();
+  const curruntProjectId = localStorage.getItem("curruntProjectId");
+  const [isLoading, setIsLoading] = useState(false);
+
+  const onSubmit = async (data: object) => {
+    setIsLoading(true);
+    const url =
+      action == "update"
+        ? `${baseUrl}/Project/${curruntProjectId}`
+        : `${baseUrl}/Project`;
+    const proccess = action == "update" ? "Updated" : "Added";
+    const method = action == "update" ? axios.put : axios.post;
```

Comment:
We can also make this as : 
```
const method =  action === "update"? "put": "post";
...
axios[method](url,data,...);
```

both are okay for me, but I feel that calling it a method and including axios in it and then calling it will make it difficult to understand the code from other people.

and having it as `axios[method]` makes it easier to read that we are just dynamically changing the method.